ask_the-users.py

```diff
@@ -0,0 +1,11 @@
+import string
+
+
+def ask_users():
+    string_=[]
+    
+    print(str1=-input('Podaj stringa'))
```

Comment:
did you test it?

Reply:
![image](https://user-images.githubusercontent.com/114613048/197198843-c0321b1d-5e4d-4480-a4cf-519a87bd50c0.png)